src/services/orderbook_service.ts

```diff
@@ -111,49 +111,15 @@ export class OrderBookService {
         const signedOrderEntities = (await this._connection.manager.find(SignedOrderEntity, {
             where: filterObject,
         })) as Array<Required<SignedOrderEntity>>;
-        let apiOrders = _.map(signedOrderEntities, orderUtils.deserializeOrderToAPIOrder);
+        const apiOrders = _.map(signedOrderEntities, orderUtils.deserializeOrderToAPIOrder);
         // Post-filters
-        apiOrders = apiOrders
-            .filter(
-                // traderAddress
-                apiOrder =>
-                    ordersFilterParams.traderAddress === undefined ||
-                    apiOrder.order.makerAddress === ordersFilterParams.traderAddress ||
-                    apiOrder.order.takerAddress === ordersFilterParams.traderAddress,
-            )
-            .filter(
-                // makerAssetAddress
-                apiOrder =>
-                    ordersFilterParams.makerAssetAddress === undefined ||
-                    orderUtils.includesTokenAddress(
-                        apiOrder.order.makerAssetData,
-                        ordersFilterParams.makerAssetAddress,
-                    ),
-            )
-            .filter(
-                // takerAssetAddress
-                apiOrder =>
-                    ordersFilterParams.takerAssetAddress === undefined ||
-                    orderUtils.includesTokenAddress(
-                        apiOrder.order.takerAssetData,
-                        ordersFilterParams.takerAssetAddress,
-                    ),
-            )
-            .filter(
-                // makerAssetProxyId
-                apiOrder =>
-                    ordersFilterParams.makerAssetProxyId === undefined ||
-                    assetDataUtils.decodeAssetDataOrThrow(apiOrder.order.makerAssetData).assetProxyId ===
-                        ordersFilterParams.makerAssetProxyId,
-            )
-            .filter(
-                // takerAssetProxyId
-                apiOrder =>
-                    ordersFilterParams.takerAssetProxyId === undefined ||
-                    assetDataUtils.decodeAssetDataOrThrow(apiOrder.order.takerAssetData).assetProxyId ===
-                        ordersFilterParams.takerAssetProxyId,
-            );
-        const paginatedApiOrders = paginationUtils.paginate(apiOrders, page, perPage);
+        const filteredApiOrders = orderUtils.filterOrders(apiOrders, ordersFilterParams);
+        // Remove expired orders
+        const dateNowSeconds = Date.now() / ONE_SECOND_MS;
+        const freshFilteredApiOrders = filteredApiOrders.filter(apiOrder =>
+            apiOrder.order.expirationTimeSeconds.gt(dateNowSeconds),
```

Comment:
could we by any chance return orders that won't expire in the next N seconds?

Reply:
What is N?

Reply:
sorry! that's my way of saying "some number that is to be determined". So, maybe `N` here could be 60?

Reply:
haha I know. I'm saying I can't move forward without a value for N.

Reply:
I would recommend at least 60, but I can defer to @steveklebanoff or @blakehendo

Reply:
Going to make it an ENV var...

Reply:
can we set this to 10 seconds for now? me and @steveklebanoff discussed this

Reply:
10 seconds works great.